### PR TITLE
Update workflow docs

### LINKED ISSUE #4255: Expand documentation on WORKFLOW in FMU Docs
FMU docs:
https://fmu-docs.equinor.com/docs/ert/reference/configuration/keywords.html#load-workflow-job

**LOAD_WORKFLOW**

Load a workflow into ERT.

**LOAD_WORKFLOW_JOB**

Load a workflow job into ERT.

**WORKFLOW_JOB_DIRECTORY**

Directory containing workflow jobs.

### PR DESCRIPTION
**Pull request by andreas-el**
**Issue**
Resolves #4255

Syncronized documentation for HOOK_WORKFLOW.
Added missing documentation parts for LOAD_WORKFLOW, LOAD_WORKFLOW_JOB, WORKFLOW_JOB_DIRECTORY.

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: docs/reference/configuration/keywords.rst [hunk 1]
```diff
@@ -2069,61 +2069,89 @@ Workflow hooks
...
+        flow of execution where you can hook in a workflow:
 
-    ::
-
-        HOOK_WORKFLOW initWFLOW        PRE_SIMULATION
-        HOOK_WORKFLOW preUpdateWFLOW   PRE_UPDATE
-        HOOK_WORKFLOW postUpdateWFLOW  POST_UPDATE
-        HOOK_WORKFLOW QC_WFLOW1        POST_SIMULATION
-        HOOK_WORKFLOW QC_WFLOW2        POST_SIMULATION
+        - Before the simulations start using :code:`PRE_SIMULATION`,
```
Comment (berland) [inline docs/reference/configuration/keywords.rst:2077]: Should we point out that by "simulations" we mean "all forward models for a realization"?

Comment (andreas-el) [reply docs/reference/configuration/keywords.rst]: Yep! Updated!

File: docs/reference/configuration/keywords.rst [hunk 2]
```diff
@@ -2069,61 +2069,89 @@ Workflow hooks
...
+        ::
+
+            LOAD_WORKFLOW  /path/to/workflow/WFLOW1
+            LOAD_WORKFLOW  /path/to/workflow/workflow2  WFLOW2
+
+        The :code:`LOAD_WORKFLOW` takes the path to a workflow file as the first
+        argument. By default the workflow will be labeled with the filename
+        internally in ERT, but you can optionally supply a second extra argument
+        which will be used as the name for the workflow.  Alternatively,
+        you can load a workflow interactively.
```
Comment (eivindjahren) [inline docs/reference/configuration/keywords.rst:2118]: What does interactively mean here?

**Review by eivindjahren — APPROVED** [review]

File: docs/reference/configuration/keywords.rst (hunk 2 above)
Comment (andreas-el) [reply docs/reference/configuration/keywords.rst:2118]: Collectively decided to not follow up on this comment.

**Pull request merged**